rubocop.yml

```diff
@@ -13,9 +13,12 @@ Lint/AmbiguousOperator:
 Lint/AmbiguousRegexpLiteral:
   Enabled: false
 Lint/AssignmentInCondition:
-  AllowSafeAssignment: false
+  AllowSafeAssignment: true
 Lint/HandleExceptions:
   Enabled: false
+Metrics/AbcSize:
+  Enabled: true
+  Max: 25
```

Comment:
The default here is 15. This metric is useful (together with `Metrics/MethodLength`) to keep methods from becoming huge; we're generally good at it now so I see no reason to remove it. And the maximum here has been raised from the default 15.